lib/langchain/tool/calculator/calculator.json

```diff
@@ -2,7 +2,7 @@
   {
     "type": "function",
     "function": {
-      "name": "calculator-execute",
+      "name": "calculator__execute",
```

Comment:
Renaming this because, apparently, Google Gemini automatically converts dashes to underscores. For example: "calculator-execute" was automatically being converted to "calculator_execute" hence the switch to double underscores to standardize across all models.